frontend/packages/ceph-storage-plugin/src/components/ocs-install/internal-mode/install-wizard-steps/select-capacity-and-nodes-step.tsx

```diff
@@ -42,6 +59,110 @@ const validate = (scName, enableMinimal, enableFlexibleScaling): ValidationType[
   return validations;
 };
 
+const isArbiterDisabled = (nodesData: NodeKind[], validateSelectedNodes?: boolean) => {
+  const validation = {
+    nodesLen: validateSelectedNodes ? 4 : 5,
+    zoneLen: validateSelectedNodes ? 2 : 3,
+  };
+  const uniqZones: Set<string> = new Set(nodesData.map((node) => getZone(node)));
+  if (nodesData.length < validation.nodesLen) return true;
+  if (uniqZones.size !== validation.zoneLen) return true;
+  const zonePerNode = countNodesPerZone(nodesData);
+  let count = 0;
+  Object.keys(zonePerNode).forEach((zone) => {
+    if (zonePerNode[zone] >= 2) count += 1;
+  });
+  return count < 2;
+};
+
+export const StretchClusterFormGroup: React.FC<stretchClusterFormGroupProps> = ({
+  state,
+  dispatch,
+}) => {
+  const { t } = useTranslation();
+  const { enableStretchCluster, selectedArbiterZone, nodes } = state;
+  const [zonesOption, setZonesOptions] = React.useState({});
+  const [showInfoAlert, setShowInfoAlert] = React.useState(true);
+  const [nodesData, nodesLoaded] = useK8sWatchResource<NodeKind[]>(nodeResource);
+
+  const nodesDataMemoized: NodeKind[] = useDeepCompareMemoize(nodesWithoutTaints(nodesData), true);
+  const uniqZones: Set<string> = new Set(nodesDataMemoized.map((node) => getZone(node)));
+
+  React.useEffect(() => {
+    dispatch({
+      type: ActionType.SET_ARBITER_VALID,
+      payload: enableStretchCluster && isArbiterDisabled(nodes, true),
+    });
+    if (nodesLoaded && isArbiterDisabled(nodesDataMemoized)) {
+      dispatch({ type: ActionType.SET_ENABLE_STRETCH_CLUSTER, payload: false });
+    }
+  }, [dispatch, enableStretchCluster, nodesDataMemoized, nodes, nodesLoaded]);
+
+  React.useEffect(() => {
+    if (nodesLoaded && enableStretchCluster) {
+      setZonesOptions(_.zipObject([...uniqZones], [...uniqZones]));
+      dispatch({ type: ActionType.SET_ARBITER_ZONE, payload: [...uniqZones]?.[0] });
+    }
+  }, [dispatch, enableStretchCluster, nodesLoaded, uniqZones]);
+  return (
+    <FormGroup fieldId="arbiter-cluster" label={t('ceph-storage-plugin~Stretch Cluster')}>
+      <Checkbox
+        aria-label={t('ceph-storage-plugin~Enable Arbiter')}
```

Comment:
You already have `label` set , its redundant to add `aria-label` here and with the same text.
```suggestion
```